Without this the build will try to download rules_java 8.12.0. we already
downloaded 8.14.0 as part of the sources, so force bazel to use that.

diff --git a/src/MODULE.bazel b/src/MODULE.bazel
index edd3af4799..de79c0decc 100644
--- a/src/MODULE.bazel
+++ b/src/MODULE.bazel
@@ -172,6 +172,11 @@
 
 register_toolchains("@androidndk//:all")
 
+bazel_dep(
+    name = "rules_java",
+    version = "8.14.0",
+)
+
 new_local_repository = use_repo_rule(
     "@bazel_tools//tools/build_defs/repo:local.bzl",
     "new_local_repository",
